### DISCUSSION TITLE
[M-Z] (9 Settembre 2019) Esercizio 6

### DISCUSSION THREAD

**Discussion by rimaout** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#482] [Esami & Esoneri M-Z]
### Traccia

<img width="561" alt="image" src="https://github.com/user-attachments/assets/c3c9bfb1-0e3d-4e70-bae9-76450d68d77f">

**Comment by matypist** [comment]
Segue una proposta di soluzione di Andrea Princic (@Princic-1837592), tratta dal suo file di soluzioni [20190909.pdf](https://raw.githubusercontent.com/Princic-1837592/metodi-matematici/refs/heads/main/soluzioni/20190909.pdf) nel suo [repository GitHub](https://github.com/Princic-1837592/metodi-matematici) contenente le sue soluzioni agli esami del prof. Piperno:

![image](https://github.com/user-attachments/assets/bf3b7f42-12a8-40fd-990d-b05f214419af)

## Tableau

![image](https://github.com/user-attachments/assets/35539901-f4db-47e2-bf6e-609f534846c7)

![image](https://github.com/user-attachments/assets/8c8528c1-7706-4d34-aee2-446d7e1c07f6)
